org.bndtools.templates.osgi/resources/templates/integration-test/bnd.bnd

```diff
@@ -12,27 +12,21 @@ Test-Cases: \${classes;CONCRETE;ANNOTATED;org.junit.runner.RunWith}
 -buildpath: \
 	\${junit},\
 	\${mockito},\
```

Comment:
we have macros here? so we should maybe have a ${bnd.testpath} macro?

Reply:
I used those two macros for now. We may introduce a new one in the future, after this first cleanup.